n2y/main.py

```diff
@@ -69,7 +69,7 @@ def main(raw_args, access_token):
     )
     parser.add_argument(
         "--database-config", default='{}',
-        help="Configuration for databases"  # TODO: Add more helpful message/description
+        help="A JSON string in the format {database_id: {sorts: {...}, filter: {...}}}. Will get passed to the Notion API directly"  # noqa: E501
```

Comment:
You can use python's built-in string concatenation to avoid the need for these long lines. E.g., see https://github.com/innolitics/n2y/pull/40/commits/91cd08b217b4a26534319a4e7202aa9dc6f8d748#diff-83f6409c23eb83ba1be05c742aba1a0718002913060149a95dbbc5186b467e8fR31-R34

Reply:
You may also want to include links to these pages:

https://developers.notion.com/reference/post-database-query-filter
https://developers.notion.com/reference/post-database-query-sort

in the help text.

Reply:
Oh nice! TIL!